packages/sdk/app-framework/src/vite-plugin/boot-loader/loader-app/boot-loader.css

```
@@ -163,39 +161,53 @@ body {
    * head sits at `(100 - arc)%` (opaque) and the tail at `100%` (transparent).
+   * NOTE: We start at -1.16deg so that we don't mask the marker.
    */
   --boot-loader-arc: 0;
   mask: conic-gradient(
-    from 0deg at 50% 50%,
+    from -1.16deg at 50% 50%,
     transparent 0,
     transparent calc((100 - var(--boot-loader-arc)) * 1%),
     #000 calc((100 - var(--boot-loader-arc)) * 1%),
     transparent 100%
   );
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)